default.nix

```diff
@@ -0,0 +1,60 @@
+{ pkgs ? (import ./nix/nixpkgs) }:
+
+let
+  source = [
+    "^.*\.md"
+    "^benchmark.*$"
+    "^examples.*$"
+    "^models.*$"
+    "^monad-bayes\.cabal$"
+    "^src.*$"
+    "^stack.*\.yaml$"
+    "^test.*$"
+  ];
+  mkMonadBayes = stackYaml: pkgs.haskell-nix.stackProject {
+    inherit stackYaml;
+    src = pkgs.lib.sourceByRegex ./. source;
+    # ghc = pkgs.buildPackages.pkgs.haskell-nix.compiler.${compiler};
+  };
+
+  # mkHaskellPackages = compiler: pkgs.haskell.packages.${compiler}.override {
+  #   overrides = self: super: {
+  #     "monad-bayes" = pkgs.haskell.lib.enableCabalFlag
+  #       (super.callCabal2nix "monad-bayes" source { }) "dev";
+  #   };
+  # };
+  # haskellPackagesGhc84 = mkHaskellPackages "ghc844";
+  # haskellPackagesGhc86 = mkHaskellPackages "ghc865";
+  # haskellPackagesGhc88 = mkHaskellPackages "ghc881";
+  # haskellPackages = haskellPackagesGhc86;
+  monad-bayes-ghc82 = mkMonadBayes "stack-ghc822.yaml";
+  monad-bayes-ghc84 = mkMonadBayes "stack-ghc844.yaml";
+  monad-bayes-ghc86 = mkMonadBayes "stack-ghc865.yaml";
```

Comment:
Does `haskell-nix` work with GHC 8.8? If so, it would be good to build against 8.8 too.

Reply:
Yes, but I'm not sure `monad-bayes` works with 8.8 right now. I did not want to have to fix it because it is outside of the scope for this PR.

Reply:
I am sure `monad-bayes` works with 8.8 right now. I've had to test this for the Stackage PR: https://github.com/commercialhaskell/stackage/pull/5189. So any failure that comes here is due to the BuildKite integration and not due to the library being incompatible with GHC 8.8.

Reply:
Thanks!